src/productcatalogservice/server.go

```diff
@@ -78,7 +85,8 @@ func init() {
 }
 
 func main() {
-	initStackDriverTracing()
+	initOpenCensus()
```

Comment:
Maybe rename OpenCensus method to be more concrete: initOpenCensusMetrics
This way it's more clear what is part of OpenTelemetry vs OpenCensus instrumentation as we work towards getting rid of OpenCensus completely.

Reply:
I just moved the helper function in there and called it initOpenCensusStats, and commented that this would be replaced with OpenTelemetry Metrics.